test/runtests.jl

```diff
@@ -87,8 +88,8 @@ end
     N = nrow(df)
     df.weights = repeat(1:5, Int(N/5))
     f = @formula(FoodExp ~ Income)
-    lm_model = lm(f, df, wts = df.weights)
-    glm_model = glm(f, df, Normal(), wts = df.weights)
+    lm_model = lm(f, df, wts = FrequencyWeights(df.weights))
```

Comment:
Better move all tests about weights to analytic_weights.jl and similar files. Also use `weights` rather than `wts`.